### DISCUSSION TITLE
Radix themes

### DISCUSSION THREAD

**Discussion by cr101** [epicweb-dev/epic-stack#384] [Q&A]
Hi,

Any plans to add the just released [Radix themes](https://www.radix-ui.com/)?

**Comment by kentcdodds** [comment]
There's already an example: https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md

I'm not yet certain whether we'll build it in. I still need to evaluate it.

**Reply by cr101** [reply]
the Radix Themes was [released 4 days ago](https://www.radix-ui.com/themes/docs/overview/releases).
Can `shadcn-custom-theme` and Radix Themes be used together?

**Reply by andrecasal** [reply]
I was actually starting to build my own Radix UI-based component library, much like shadcn but with more components, but I'm quite impressed with Radix Themes. They even have layout and typography components, which I felt where missing.

My only gripe with Radix Themes so far is that their scaling system is not aligned with TailwindCSS 😭
<img width="1719" alt="Screenshot 2023-09-02 at 12 38 33" src="https://github.com/epicweb-dev/epic-stack/assets/22457463/07cf6ca8-1db6-4732-87a4-c3e3172cafb4">

**Reply by andrecasal** [reply]
Just found my deal breaker for Radix Themes.

Here's how Radix Themes works:
1. Import a huge (1000+ lines) CSS file in `root.tsx` 🤢
2. Style your component using a CSS file 🤮
3. Import that CSS file in your component and so you can use that classname 😭

Here's [what I'm building](https://andrecasal.com/ui):
1. Small CSS file with CSS variables to control your entire theme 🎊
2. Custom utility classes in tailwind.config.ts that use those CSS vars  🎉
3. Components use those custom utility classes which are available anywhere in your app 🥳
